models/setting.js

```diff
@@ -38,20 +38,19 @@ SettingSchema.statics.getSettings = function () {
 };
 
 /**
- * Gets the moderation settings and sends it back
+ * Gets the settings visible to the public
  * @return {Promise} moderation the settings for how to moderate comments
  */
-SettingSchema.statics.getModerationSetting = function () {
-  console.log('Getting moderation setting');
-  return this.findOne({id: '1'}).select('moderation');
+SettingSchema.statics.getPublicSettings = function () {
+  return this.findOne({id: '1'}).select('moderation infoBoxEnable infoBoxContent');
```

Comment:
This is good while settings are simple. I forsee breaking settings, their defaults and which are public/private into a config situation in the future.